### PR TITLE
Include new illustrations with the upcoming colour schema changes

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
#### Summary

Include new illustrations with the upcoming new colour schema changes

#### Description

In this PR we include new illustrations using the upcoming new colour schema.

We also adapt the components using those illustrations to support rendering the current and the new ones based on the selected theme.

### CODE REVIEW

**Comment by CarlosCortizasCT** [comment]
We're waiting for some inconsistencies we found and communicated to the design team.

File: packages/assets/images/doors-closed-rebranding.svg
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply packages/assets/images/doors-closed-rebranding.svg:1]: Yes.
My idea is, after the rollout, we just overwrite the old files with the new ones and we delete all the ones with the _rebranding_ suffix.

**Review by kark — APPROVED** [review]

**Review by ddouglasz — APPROVED** [review]
Thank you 🙌🏾

File: packages/assets/images/doors-closed-rebranding.svg
[the remark this answers is not in the document]
Comment (emmenko) [reply packages/assets/images/doors-closed-rebranding.svg:1]: Perfect, thanks!

**Review by emmenko — APPROVED** [review]

**Review by Rhotimee — APPROVED** [review]

**Pull request merged**